js/initcontext.go

```diff
@@ -239,7 +240,8 @@ func (i *InitContext) Open(ctx context.Context, filename string, args ...string)
 	}
 
 	if len(args) > 0 && args[0] == "b" {
-		return i.runtime.ToValue(data), nil
+		b := i.runtime.NewArrayBuffer(data)
+		return &b, nil
```

Comment:
Reading https://pkg.go.dev/github.com/dop251/goja#ArrayBuffer I would argue you should be calling ToValue on the value and just returning `goja.Value` instead of `interface{}` above. 

This should be exactly what happens either way but IMO it is better for it to be explicit.

Reply:
Resolved by 46fffe7e.